File: Zadania/2025/Zadanie-7/Python/Bron.py
class Bron():
    def __init__(self , nazwaBroni , czyAutomat = False):
        self.nazwaBroni = nazwaBroni
        self.czyAutomat = czyAutomat
    def shoot(self):
        if self.czyAutomat:
            print(f"{self.nazwaBroni} RATATAATTATAT!")
        else :
            print(f"{self.nazwaBroni} BOOM!")

File: Zadania/2025/Zadanie-7/Java/Bron.java
package Java;

public class Bron {
    String nazwaBroni;
    boolean czyAutomat;

    Bron(String nazwaBroni, boolean czyAutomat) {
        this.nazwaBroni = nazwaBroni;
        this.czyAutomat = czyAutomat;
    }

    public void shoot() {
        if (czyAutomat) {
            System.out.println(nazwaBroni + " RATATATATATAT!");
        } else {
            System.out.println(nazwaBroni + "BOOM!");
        }
    }
}
